radars/SRE.csv

```diff
@@ -0,0 +1,951 @@
+<h3>Pros</h3>
+<ul>
+<li>Fully managed</li>
+<li>Simple to setup and get started</li>
+<li>Direct integration into other services</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Api gateway can be more expensive when dealing with large loads</li>
+<li>Difficult to have large application cohesion</li>
+</ul>
+<p>When decision made: 5/1/2020</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Serverless.com,Infrastructure,Hold,none,"<p><a href=""https://serverless.com/"">Serverless</a> The Serverless Framework gives you everything you need to develop, deploy, monitor and secure serverless applications on any cloud.</p>
+<h3>Pros</h3>
+<ul>
+<li>Cloud agnostic</li>
+<li>Easy to use tooling</li>
+<li>Well documented</li>
+<li>Can save significant amounts of money over traditional monolithic applications</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Path to serverless is generally a rewrite</li>
+<li>Other applications can be dockerized and ran on existing resources</li>
+</ul>
+<p>When decision made: 5/1/2020</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Chef (All),Infrastructure,Divest,none,"<p><a href=""https://www.chef.io/"">Chef (All)</a> automates infrastructure configuration, ensuring every system is configured correctly and consistently.</p>
+<h3>Pros</h3>
+<ul>
+<li>Manage your infrastructure declaratively</li>
+<li>Drift detection</li>
+<li>Easy to replicate environments</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Lack of community support</li>
+<li>Not open source</li>
+<li>Steep learning curve</li>
+<li>Requires scripting for very common tasks</li>
+</ul>
+<p>Relatively new addition to the infrastructure as code arena but <a href="""">Terraform</a> or <a href="""">Cloud Formation</a> are strongly preferred.</p>
+<p>When decision made: 5/1/2020</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+ECS,Infrastructure,Divest,none,"<p><a href=""https://aws.amazon.com/ecs/"">Elastic Container Service (ECS)</a> is a fully managed container orchestration service. Customers such as Duolingo, Samsung, GE, and Cookpad use ECS to run their most sensitive and mission critical applications because of its security, reliability, and scalability.</p>
+<h3>Pros</h3>
+<ul>
+<li>Support for serverless containers through Fargate</li>
+<li>AWS native</li>
+<li>Managed infrastructure</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Lacks community support</li>
+<li>Only works with AWS</li>
+</ul>
+<p>While ECS and Kubernetes are very similar <a href="""">Kubernetes</a> is favored heavily.</p>
+<p>When decision made: 5/1/2020</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Datadog,Observability,Hold,none,"<p><a href=""https://www.datadoghq.com/"">Datadog</a> is a hosted platform for monitoring and analytics. Datadog enables easy instrumentation of many types of applications and infrastructure.</p>
```

Comment:
Is this Hold?  Who uses it?

Reply:
There are other teams that use it but some have been told not to use it.